src/components/spreadsheet/EnhancedRCSASpreadsheet.tsx

```
@@ -0,0 +1,825 @@
+        }
+        break;
+      case 'Delete':
+      case 'Backspace':
+        if (columns[col].editable && !e.metaKey && !e.ctrlKey) {
+          e.preventDefault();
+          startEditing('');
+        }
+        break;
+      case 'Tab':
+        e.preventDefault();
+        const nextCol = e.shiftKey ? col - 1 : col + 1;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)